### DISCUSSION TITLE
Refresh Data with Fetch

### DISCUSSION THREAD

**Discussion by ensargunesdogdu** [TanStack/table#2441] [General]
Greetings everyone,

I was trying to implement a table which fetch data from backend and do some filters. Then i want to refreshed content from backend to get last results. I used code below as template.

https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/kitchen-sink?file=/src/App.js

I changed the part below. to get data from backend. When i filter some data and click on reload button. My filters disappear :(

Could you please help me understand why ?
```
const [data, setData] = React.useState([])
// const [originalData] = React.useState(xxxx)
function loadData() {
        fetch(backendURL, {
            method: 'GET',
            headers: {
                Accept: 'application/json',
                'Content-Type': 'application/json',
                Authorization: 'Bearer ' + this._token
            }
        }).then(res => {
            return res.json();
        }).then((data)=>{
            setData(data);
        })
    }

const resetData = () => {
       // Don't reset the page when we do this
       skipResetRef.current = true
       loadData();
}
```

**Comment by gargroh** [comment]
check `autoResetFilters: Boolean` at https://github.com/tannerlinsley/react-table/blob/master/docs/api/useFilters.md
